boards/arm/rcar_h3_salvatorx/doc/rcar_h3_salvatorx.rst

```diff
@@ -0,0 +1,188 @@
+
+- `Renesas R-Car H3 chip`_
+
+Supported Features
+==================
+
+Here is the current supported features when running Zephyr Project on the R-Car Salvator-X CR7:
+
++-----------+------------------------------+--------------------------------+
+| Interface | Driver/components            | Support level                  |
++===========+==============================+================================+
+| PINCTRL   | pinctrl                      |                                |
++-----------+------------------------------+--------------------------------+
+| CLOCK     | clock_control                |                                |
++-----------+------------------------------+--------------------------------+
+| GPIO      | gpio                         |                                |
++-----------+------------------------------+--------------------------------+
+| UART      | uart                         | serial port-polling            |
++           +                              +                                +
+|           | FT232RQ / CP2102             | serial port-interrupt          |
++-----------+------------------------------+--------------------------------+
+| CAN       | can                          | normal mode                    |
++           +                              +                                +
+|           | TCAN332GDCNT                 | loopback mode                  |
++-----------+------------------------------+--------------------------------+
+| I2C       | i2c                          | interrupt driven               |
++-----------+------------------------------+--------------------------------+
+
+It's also currently possible to write on the ram console.
+
+Connections and IOs
+===================
+
+.. figure:: img/r-car-h3-salvator-x-connections.jpg
+   :align: center
+   :alt: R-Car Salvator-X connections
+
+GPIO
+----
+
+By running Zephyr on H3 Salvator-X, the software readable push buttons 'SW20',
+'SW21', 'SW22' can be used as input, and the software contollable LEDs 'LED4',
+'LED5', 'LED6' can be used as output.
+
+UART
+----
+
+Salvator-X board is providing two serial ports:
+
+- one is for A53/A57 processors
+- the other one is for CR7
+
+Both ports are converted to USB through CP2102 converters and they are exposed
+as follows:
+
++-----------+-----------+
+| Connector | Processor |
++===========+===========+
+| CN25      | A53/A57   |
++-----------+-----------+
+| CN26      | CR7       |
++-----------+-----------+
+
+Programming and Debugging
+*************************
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Supported Debug Probe
+=====================
+
+The "Olimex ARM-USB-OCD-H" probe is the only officially supported probe. This
+probe is supported by OpenOCD that is shipped with the Zephyr SDK.
+
+The "Olimex ARM-USB-OCD-H" probe needs to be connected with a SICA20I2P adapter
```

Comment:
You don't need the SICA20I2P adapter to connect a debug probe on this board !

Reply:
You're right: I changed the name of the connector, but I forgot to remove that part